src/context/UserContext.tsx

```diff
@@ -0,0 +1,7 @@
+import { UserProvider } from '@auth0/nextjs-auth0/client';
+
+const UserContext = ({ children }) => {
+  return <UserProvider>{children}</UserProvider>;
+};
```

Comment:
Is this component still relevant? Since `UserProvider` looks like it can be used by itself.